src/test/java/com/powsybl/openloadflow/network/LfNetworkTest.java

```diff
@@ -204,4 +204,24 @@ void testGraphViz() throws IOException {
         network.getLine("NHV1_NHV2_1").getTerminal1().disconnect();
         testGraphViz(network, false, "sim1_disconnected.dot");
     }
+
+    @Test
+    void testDisabledVoltageControl() {
+        Network network = VoltageControlNetworkFactory.createWithDependentVoltageControls();
+        List<LfNetwork> lfNetworks = Networks.load(network, new MostMeshedSlackBusSelector());
+        assertEquals(1, lfNetworks.size());
+        LfNetwork lfNetwork = lfNetworks.get(0);
+        lfNetwork.getZeroImpedanceNetworks(LoadFlowModel.AC); // to update.
+        LfBus b1 = lfNetwork.getBusById("b1_vl_0");
+        assertEquals(VoltageControl.MergeStatus.MAIN, b1.getGeneratorVoltageControl().orElseThrow().getMergeStatus());
+        LfBus b2 = lfNetwork.getBusById("b2_vl_0");
+        assertEquals(VoltageControl.MergeStatus.DEPENDENT, b2.getGeneratorVoltageControl().orElseThrow().getMergeStatus());
+        LfBus b3 = lfNetwork.getBusById("b3_vl_0");
+        assertEquals(VoltageControl.MergeStatus.DEPENDENT, b3.getGeneratorVoltageControl().orElseThrow().getMergeStatus());
+        lfNetwork.getBusById("b01_vl_0").setDisabled(true); // only g1
+        assertFalse(b1.getGeneratorVoltageControl().orElseThrow().isDisabled());
+        assertFalse(b1.getGeneratorVoltageControl().orElseThrow().isHidden());
+        assertTrue(b2.getGeneratorVoltageControl().orElseThrow().isHidden()); // FIXME
```

Comment:
Fix by 
`    public boolean isHidden() {
        // collect all voltage controls with the same controlled bus as this one and also all voltage controls coming
        // from merged ones
        List<VoltageControl<?>> voltageControls = findVoltageControlsSortedByPriority(controlledBus);
        if (voltageControls.isEmpty()) {
            return true; // means all disabled
        } else {
            // we should normally have max 3 voltage controls (one of each type) because already merged
            return voltageControls.get(0) != this && voltageControls.get(0).getMainVoltageControl() != this.getMainVoltageControl();
        }
    }`